src/store/reducers.js

```diff
@@ -52,7 +53,13 @@ function entityLoaded(state, { payload: { entitlements } } = { payload: {} }) {
                 title: 'Patch',
                 name: 'patch',
                 component: PatchTab
+            },
+            isEntitled(entitlements && entitlements.insights) && {
```

Comment:
Is the `ROS` enabled for all users? If not we might want to add the cookie check as well here.

Reply:
Ros is not enabled for all users. Added cookie check.

Reply:
@karelhala,
I forgot to mention one thing earlier that ROS is not enabled by default like few other services and designed for RHEL instances from cloud (AWS/azure). 
Along with cookie check, do I need to add extra conditions here? Or it is ok to show Tab with empty recommendations table.